src/arm/cpu.rs

```diff
@@ -0,0 +1,30 @@
+use cfg_if::cfg_if;
+use libc;
+
+pub const DAV1D_ARM_CPU_FLAG_NEON: libc::c_uint = 1 << 0;
```

Comment:
Where is this used? Do those places import it from here?

Reply:
This is used right below here in `dav1d_get_cpu_flags_arm`. It's also then checked in some other files, though that code hasn't been added in yet.

Reply:
Sounds good, just wanted to make sure those other places use this definition too.